storey/flow.py

```diff
@@ -170,11 +174,38 @@ async def _run_loop(self):
         try:
             for path in self._paths:
                 async with aiofiles.open(path, mode='r') as f:
-                    if self._skip_header:
-                        await f.readline()
+                    header = None
+                    if self._with_header:
+                        line = await f.readline()
+                        if self._build_dict:
+                            header = next(csv.reader([line]))
```

Comment:
what if the csv contains a header, but you don't want to build a dict. you just ignore the header?

Reply:
Yes, same as before this PR. What else would you do with it?

Reply:
oh for some reason i thought you sent it downstream as well..
but maybe worth sending it downstream as well (if not specified otherwise), the user might want to do his own manipulations/logic on that 🤔

Reply:
The header? Definitely not send it downstream. If the user really wants that for some reason, they can set or leave `with_header=False`.